### ISSUE TITLE
Slack Bot Not Updating Message

### ISSUE THREAD

**Issue by connor-tyndall** [ogrodnek/code-pipeline-slack#4]
The Slack Bot I have configured, it is not updating (`chat.update`) the original message. It is posting a new message for each change in my pipeline. Your gif shows that it should be updating the original message as the steps are being executed in the pipeline. 

Please advise.

**Comment by rayj-pgi** [comment]
Hey Connor, Did you add the permission scope to your bot?

> It is required to add the permission scope 'Access user’s public channels' (channels:history).

**Comment by connor-tyndall** [comment]
Yes, and it posting to the channel. Just not updating the messages. I should mention this is a private Slack channel (if that makes a difference). 

<img width="342" alt="image" src="https://user-images.githubusercontent.com/24273471/50615469-abc08800-0eb2-11e9-93fc-883b93116100.png">

**Comment by connor-tyndall** [comment]
So that's what it was... it works if I use a PUBLIC slack channel.

**Comment by connor-tyndall** [comment]
However, when I click on `Build Logs` it takes me to a `null` logGroup and `null` logStream. 

<img width="999" alt="image" src="https://user-images.githubusercontent.com/24273471/50616955-d3b2ea00-0eb8-11e9-98a9-760e6676c4f5.png">

**Comment by theodorosidmar** [comment]
Well, I do not know if this suits to this issue, but the final result is far away from the result shown at `build.gif` on `readme.md` 🙁

Result:
![image](https://user-images.githubusercontent.com/18275739/59526588-a464c800-8eaf-11e9-8b4a-5895500ff902.png)

Gif:
![image](https://github.com/ogrodnek/code-pipeline-slack/blob/master/build.gif?raw=true)

**Comment by allantaylor8907** [comment]
@rayjanoka or @connor-tyndall 
I can't get the bot to post anything at all. I have the cloudformation stack deployed with the master branch of repo instead of the outdated deployment link. I also added in:
`  SLACK_CHANNEL: !Ref SlackChannel` 
I have both tokens set but no messaging ever appears. Curious if you could point me in the right direction to test the lambda
